pages/linux/dex.md

```diff
@@ -0,0 +1,32 @@
+# dex
+
+> DesktopEntry Execution is a program to generate and execute DesktopEntry files of the Application type.
+> More information: <https://github.com/jceb/dex>.
+
+- Perform an autostart/execute all programs in the autostart folders:
+
+`dex --autostart`
+
+- Perform an autostart/execute all programs in the specified folders:
+
+`dex --autostart --search-paths {{path/to/directory}}:{{path/to/directory}}`
+
+- Preview the programs would be executed in a regular autostart:
+
+`dex --autostart --dry-run`
+
+- Preview the programs would be executed in a GNOME specific autostart:
```

Comment:
When showing a new option in an example and it doesn't have to be combined with another, I would just use the new one. So `dex --autostart --environment GNOME`

Reply:
Suggestion?

Reply:
```
- Execute all autostart programs in the GNOME environment:

`dex --autostart --environment {{GNOME}}`
```
Also, I'd put that above the `--dry-run` example.